File: app/src/main/java/es/etologic/mahjongscoring2/app/game/game_list/GameListModule.java
package es.etologic.mahjongscoring2.app.game.game_list;

import android.content.Context;

import dagger.Module;
import dagger.Provides;

@Module
public class GameListModule {

    @Provides
    GameListRvAdapter provideGameListRvAdapter(Context context) {
        return new GameListRvAdapter(context);
    }
}

File: app/src/main/java/es/etologic/mahjongscoring2/app/game/game_list/GameListModule.kt
package es.etologic.mahjongscoring2.app.game.game_list

import android.content.Context
import dagger.Module
import dagger.Provides

@Module
class GameListModule {
    
    @Provides
    internal fun provideGameListRvAdapter(context: Context): GameListRvAdapter {
        return GameListRvAdapter(context)
    }
}
